lang/strings_english.txt

```diff
@@ -1184,6 +1187,7 @@ $s_resolved = 'Resolved';
 $s_total = 'Total';
 $s_balance = 'Balance';
 $s_most_active = 'Most Active';
+$s_most_votes = 'Hot';
```

Comment:
Hot is a vague term. Is it issues with most comments, most votes, etc.  Let's use a term that is consistent with the criteria we are using.